packages/app-next/src/App.tsx

```diff
@@ -49,7 +50,7 @@ TODO:
 /* app.tsx */
 
 const app = createApp({
-  plugins: [graphiqlPlugin, pagesPlugin, techRadarPlugin],
+  plugins: [graphiqlPlugin, pagesPlugin, techRadarPlugin, techdocsPlugin],
```

Comment:
I believe this is not necessary, installing `@backstage/plugin-techdocs`package should be enough:

```suggestion
  plugins: [graphiqlPlugin, pagesPlugin, techRadarPlugin],
```